evolver/device.py

```diff
@@ -71,6 +71,10 @@ def schema(self):
             "controllers": [{"kind": str(type(a)), "config": a.Config.model_json_schema()} for a in self.controllers],
         }
 
+    @property
+    def active(self):
+        return self.enable_control and self.enable_commit
```

Comment:
I'm just wondering how this might intuitively get interpreted by someone manually controlling the hardware, i.e., not using ``control`` but committing commands?

"active" sounds to me like they should be able to but the above implies that you shouldn't/can't. We could have a "running" status flag, however at that point, you literally just have these two flags. Perhaps it's best to nuke this state reduction and just natively report both ``enable_control`` and ``enable_commit`` in the ``healthz`` resp?

Reply:
Really lack of `commit` is what should make it inactive, but no control would generally mean nothing to `commit`.

But also I wonder if maybe these needn't be separate. Perhaps I can just make `enable_control` mean both execute control and commit - the separation is probably not too useful...

Reply:
>Really lack of commit is what should make it inactive, but no control would generally mean nothing to commit.

Exactly.

>But also I wonder if maybe these needn't be separate. Perhaps I can just make enable_control mean both execute control and commit - the separation is probably not too useful...

Do you mean separate at all or just as reported from ``healthz``? 

With the way we have the configuration setup, I def think it's needed to have both. One deactivates hardware control (``enable_commit``) and the other starts the controller(s)/experiment running (``enable_control``). Removing ``enable_control`` would require someone who wants to pause an experiment (control) and manually interact with the hardware to have to stop the running experiment by creating a new config with the ``control`` section omitted. Seems burdensome to me, no?

Reply:
>>Really lack of commit is what should make it inactive, but no control would generally mean nothing to >commit.
>
>Exactly.

Actually, I'm not sure if that's entirely accurate as ``enable_commit`` is only checked in ``evolver.device.Evolver.loop_once`` and not at any communication layer. So a user could programmatically control the hardware still from the SDK (but not the app).